### PR TITLE
add feature/array

### PR DESCRIPTION
**Pull request by sergicast**

### CODE REVIEW

File: staff/sergi-ramos/arrays/arroz/arroz.prototype.indexOf.spec.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+describe("Arroz.prototype.indexOf", function () {
+    it("should return the index of array when find the first element that you pass like a parameter", function () {
+        var array = new Arroz(1, 2, 3, 4, 5);
+        
+
+        var a =  array.indexOf(2)
+
+       expect(a).toBe(true);
```
Comment (alexiparkhio) [inline staff/sergi-ramos/arrays/arroz/arroz.prototype.indexOf.spec.js:8]: You are barely testing anything here. Each "it" test should have several expectations covering every possibility. Saying `a to be true` is only proving that the variable `a` is defined but nothing else.

Some suggestions for this test:

`expect(a).toBe(1)`
`it('should not modify the original arroz in any way' function () {})`

Comment (alexiparkhio) [reply staff/sergi-ramos/arrays/arroz/arroz.prototype.indexOf.spec.js:8]: Also, be careful: the file name is `indexOf` and camel case for file names is not allowed. Change it to: `index-of`

Comment (sergicast) [reply staff/sergi-ramos/arrays/arroz/arroz.prototype.indexOf.spec.js]: i've change the name of the file and apply the changes you suggested me, thanks!

**Comment by alex-martinez-jativa** [comment]
Good job Sergi, but you should remove unnecessary white lines in your code, to improve your clean code

File: staff/sergi-ramos/arrays/array-functions/every.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+function every(array, callback) { debugger
```
[the remark this answers is not in the document]
Comment (sergicast) [reply staff/sergi-ramos/arrays/array-functions/every.js:2]: done! thanks! ;)

**Pull request merged**